src/test/java/com/gradle/enterprise/conventions/customvalueprovider/CICustomValueProviderIntegrationTest.java

```diff
@@ -33,7 +33,7 @@ public void workWithJenkins() {
         succeeds("help");
 
         assertTrue(getConfiguredBuildScan().containsLink("Jenkins Build", "https://jenkins"));
-        assertTrue(getConfiguredBuildScan().containsValue("Build ID", "jenkins_id"));
+        assertTrue(getConfiguredBuildScan().containsValue("Build_ID", "jenkins_id"));
```

Comment:
```suggestion
        assertTrue(getConfiguredBuildScan().containsValue("buildId", "jenkins_id"));
```

Shouldn't this also use the new value?